packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -359,6 +359,20 @@ const componentFiles = async (
   const intForeignKeys = intForeignKeysForModel(model)
   let fileList = {}
   const componentMetadata = {
+    Enum: {
+      componentName: 'RadioField',
+      defaultProp: 'defaultChecked',
+      validation: () => false,
```

Comment:
Should we also support `isRequired` here as we do for some other types?

Reply:
I removed the required-ness because the form validation was getting confused when the enum wasn't in the form names .... but that was for the enum list ... I bet just single enum can.

Reply:
It was hard to test not required for a single enum since the radio button is always checked.

Reply:
For unrequired enums I think it's important that no radio button is selected from the start, and that it's possible to go back to the "not selected" state (just like a unrequired text input is empty from the start, and it's easy to go back to the empty state even after you've provided a value). Another way to handle this is to have an extra radio button: "Not provided"

Reply:
> Another way to handle this is to have an extra radio button: "Not provided"

Yeah, I'm not sure how to clean the radio state even if there is a selected item (one that was previously saved).

I think the "None" or "N/A" option may be best.

So, if not required add that option or checkbox and set to "" or null or something that sets the field value to null in the db.